### PR TITLE
1056 - Fix on X Button Alignment Issue on Tabs (IPHONE X) 3.0

### LINKED ISSUE #1056: Tabs: On mobile, alignment issue on X button of dismissible tab with dropdown
**Describe the bug**
Tabs: On mobile, alignment issue on X button of dismissible tab with dropdown .

**To Reproduce**
1. Go to http://localhost:4000/components/tabs/test-dismissibles-with-dropdowns.html on Android or iOS devices
2. Click the arrow down on the right-most part of tab
3. Notice that the X button is out of position in "Plastic (Dismissible & Dropdown)" tab

**Expected behavior**
X button in the dismissible tab with dropdown should be in the right position.

**Screenshots**
screen shot 2018-10-23 at 6 53 24 pm

**Platform**
 - Android and iOS devices

**Additional context**
No additional context.

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Tweaking the position of "X" Button.

**Related github/jira issue (required)**:

Closes #1056 

**Steps necessary to review your pull request (required)**:

open ios simulator
go to http://localhost:4000/components/tabs/test-dismissibles-with-dropdowns.html
go to  http://localhost:4000/components/tabs/example-dismissible-tabs.html

### CODE REVIEW

File: src/components/popupmenu/_popupmenu.scss [hunk 1]
```diff
@@ -27,6 +27,42 @@
   }
 
   &.bottom {
+    .popupmenu {
+      .icon {
+        &.close {
+        padding-left: 6px;
```
Comment (ericangeles) [inline src/components/popupmenu/_popupmenu.scss]: Looks good @tjamesallen15. Just need to add some spacing to all css properties that you created. Install scss-lint (style lint) so you can easily track css errors.

Comment (ericangeles) [reply src/components/popupmenu/_popupmenu.scss]: err

**Comment by tmcconechy** [comment]
In addition to the comments (the lint errors).
Also merge master on and it should fix the test issue.

**Comment by tjamesallen15** [comment]
Hi @tmcconechy  and @ericangeles  this is already updated! thank you!!

**Comment by ericangeles** [comment]
I got a 🍌for you my friend @tjamesallen15. Cool beans!

**Pull request merged**